CMakeLists.txt

```diff
@@ -0,0 +1,117 @@
+cmake_minimum_required(VERSION 3.9)
```

Comment:
yo, do this like, work? I couldn't get CMake to work with the libraries that Alla had added when I was fighting with it yesterday